frontend/src/metabase/components/Popover.jsx

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 import ReactDOM from "react-dom";
+import SandboxedPortal from "metabase/components/SandboxedPortal";
```

Comment:
Is `<SandboxedPortal>` something that we'd use outside of these lower level components? Not a huge deal but if not it feels like there might be a better directory for that to live in. It's not strictly enforced but my mental model at least is that stuff in components would be used on a semi regular basis.

Reply:
I agree that `SandboxedPortal` sitting next to UI components like `Calendar` doesn't seem quite right but I couldn't think of a better place in the current folder structure. One thing it _does_ have going for it is that it is (besides react/react-dom) dependency-free, so maybe we can make some additional subfolders to the components folder soon. Not sure which one this would live in yet.